File: 3846-minimum-operations-to-make-array-sum-divisible-by-k/minimum-operations-to-make-array-sum-divisible-by-k.py
class Solution:
    def minOperations(self, nums: List[int], k: int) -> int:
        total = sum(nums)
        # if total % k == 0:
        #     return 0
        # if total < k:
        #     return total
        count = 0
        while total % k != 0:
            count += 1
            total -= 1
        return count

File: 3846-minimum-operations-to-make-array-sum-divisible-by-k/minimum-operations-to-make-array-sum-divisible-by-k.cpp
class Solution {
public:
    int minOperations(vector<int>& nums, int k) {
        int total = 0;
        for (int x: nums){
            total += x;
        }
        int count = 0;
        while (total % k != 0){
            count += 1;
            total -= 1;
        }
        return count;
    }
};